python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -950,17 +950,28 @@ def fetch_fivetran_workspace_data(
                     )
                     continue
 
-                schema_config_details = client.get_schema_config_for_connector(
-                    connector_id=connector.id
-                )
-                schema_config = FivetranSchemaConfig.from_schema_config_details(
-                    schema_config_details=schema_config_details
-                )
+                try:
+                    schema_config_details = client.get_schema_config_for_connector(
+                        connector_id=connector.id
+                    )
+                except Failure as e:
+                    if "404" in e.description:
```

Comment:
this is kinda sketchy -- is it possible to push this handling down into `get_schema_config_for_connector` and operate on the raw `Response` object?

Just seems like this isn't a super robust way of differentiating 404 errors from other error types

Reply:
Fair enough. Changes made in [0146a61](https://github.com/dagster-io/dagster/pull/29881/commits/0146a61e917fe9982eedeef7ad9329fecc2cd6f0)